src/lib/components/Feedback/index.js

```diff
@@ -85,112 +143,8 @@ class Feedback extends BaseElement {
-                  aria-labelledby="helpful-label no-label"
-                  name="Helpfulness"
-                  value="0"
-                />
-              </label>
-            </div>
-
-            <div class="web-feedback__row">
-              <div id="effective-label">
-                Did this page help you complete your goal(s)?
-              </div>
-              <label>
-                <input
-                  type="radio"
-                  aria-labelledby="effective-label yes-label"
-                  name="Effectiveness"
-                  value="1"
-                />
-              </label>
-              <label>
-                <input
-                  type="radio"
-                  aria-labelledby="effective-label no-label"
-                  name="Effectiveness"
-                  value="0"
-                />
-              </label>
-            </div>
-
-            <div class="web-feedback__row">
-              <div id="complete-label">
-                Did this page have the information you needed?
-              </div>
-              <label>
-                <input
-                  type="radio"
-                  aria-labelledby="complete-label yes-label"
-                  name="Completeness"
-                  value="1"
-                />
-              </label>
-              <label>
-                <input
-                  type="radio"
-                  aria-labelledby="complete-label no-label"
-                  name="Completeness"
-                  value="0"
-                />
-              </label>
-            </div>
-
-            <div class="web-feedback__row">
-              <div id="accuracy-label">
-                Was this page's information accurate?
-              </div>
-              <label>
-                <input
-                  type="radio"
-                  aria-labelledby="accuracy-label yes-label"
-                  name="Accuracy"
-                  value="1"
-                />
-              </label>
-              <label>
-                <input
-                  type="radio"
-                  aria-labelledby="accuracy-label no-label"
-                  name="Accuracy"
-                  value="0"
-                />
-              </label>
-            </div>
-
-            <div class="web-feedback__row">
-              <div id="readable-label">Was this page easy to read?</div>
-              <label>
-                <input
-                  type="radio"
-                  aria-labelledby="readable-label yes-label"
-                  name="Readability"
-                  value="1"
-                />
-              </label>
-              <label>
-                <input
-                  type="radio"
-                  aria-labelledby="readable-label no-label"
-                  name="Readability"
-                  value="0"
-                />
-              </label>
-            </div>
+            ${this.renderQuestions(additionalQuestionsToDisplay)}
```

Comment:
nit, can probably rename to `additionalQuestion` to match `defaultQuestions`

Reply:
`additionalQuestions` is used already on line 20 though 😬

Imma merge it in for now, and if we come up with a different naming we can always change it hopefully quickly.